src/status.rs

```diff
@@ -111,31 +114,40 @@ pub async fn is_finalized(sentry: &SentryApi, campaign: &Campaign) -> Result<IsF
 
 pub async fn get_status(sentry: &SentryApi, campaign: &Campaign) -> Result<Status, Error> {
     // continue only if Campaign is not Finalized
+    let leader = campaign.channel.spec.validators.leader();
     let leader_la = match is_finalized(sentry, campaign).await? {
         IsFinalized::Yes { reason, balances } => return Ok(Status::Finalized(reason, balances)),
         IsFinalized::No { leader } => leader,
     };
+    let leader_la_clone = match is_finalized(sentry, campaign).await? {
```

Comment:
Cloned these two variables since I couldn't find out how to reuse properties from the original one. There's probably a better way